homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
-        name="UPS Shutdown Delay",
     ),
     "dwake": SensorEntityDescription(
         key="dwake",
         translation_key="wake_delay",
-        name="UPS Wake Delay",
     ),
     "end apc": SensorEntityDescription(
         key="end apc",
         translation_key="date_and_time",
-        name="UPS Date and Time",
         entity_registry_enabled_default=False,
     ),
     "extbatts": SensorEntityDescription(
         key="extbatts",
         translation_key="external_batteries",
-        name="UPS External Batteries",
     ),
     "firmware": SensorEntityDescription(
         key="firmware",
         translation_key="firmware_version",
-        name="UPS Firmware Version",
         entity_registry_enabled_default=False,
     ),
     "hitrans": SensorEntityDescription(
         key="hitrans",
-        name="UPS Transfer High",
+        translation_key="transfer_high",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
     ),
     "hostname": SensorEntityDescription(
         key="hostname",
         translation_key="hostname",
-        name="UPS Hostname",
         entity_registry_enabled_default=False,
     ),
     "humidity": SensorEntityDescription(
         key="humidity",
-        name="UPS Ambient Humidity",
+        translation_key="humidity",
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.HUMIDITY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "itemp": SensorEntityDescription(
         key="itemp",
-        name="UPS Internal Temperature",
+        translation_key="internal_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "laststest": SensorEntityDescription(
         key="laststest",
         translation_key="last_self_test",
-        name="UPS Last Self Test",
     ),
     "lastxfer": SensorEntityDescription(
         key="lastxfer",
         translation_key="last_transfer",
-        name="UPS Last Transfer",
         entity_registry_enabled_default=False,
     ),
     "linefail": SensorEntityDescription(
         key="linefail",
         translation_key="line_failure",
-        name="UPS Input Voltage Status",
     ),
     "linefreq": SensorEntityDescription(
         key="linefreq",
-        name="UPS Line Frequency",
+        translation_key="line_frequency",
         native_unit_of_measurement=UnitOfFrequency.HERTZ,
         device_class=SensorDeviceClass.FREQUENCY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "linev": SensorEntityDescription(
         key="linev",
-        name="UPS Input Voltage",
+        translation_key="line_voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
```

Comment:
These 2 can make use of device class translations, so translation keys can be removed